### ISSUE TITLE
Lack of maintenance and inviting to fork

### ISSUE THREAD

**Issue by Utopiah** [jeromeetienne/AR.js#374]
Based on : 
- no answer from project owner regarding maintenance here #356, via direct email or on [Twitter](https://twitter.com/utopiah/status/960823123431051264)
- unfruitful discussion of moving to a Github organisation #253
- 29 PRs waiting as of today
- last commit nearly 2 months old during which dozens issues piled up

I have no specific interest in this as I am not working on anything relying on this project. That being said from a bigger picture perspective I do feel it's a shame that there is such a large interest in this project, past but also importantly on-going efforts to work with it. Due to the time required to have a proper implementation of the WebXR specifications for WebAR this allows for a smoother transition.

I invite the community of people who are relying on this project to fork it else I'm afraid the project will inexorably die.

**Comment by donmccurdy** [comment]
I think the main question should be, are there volunteers willing and able to maintain the project, or at least review and test PRs periodically? Assuming that is the case, hopefully Jerome can clarify whether he'd prefer that they fork the project, take collaborator roles here, or move to a GitHub organization.

To quantify current state of things:

![screen shot 2018-06-27 at 3 02 01 pm](https://user-images.githubusercontent.com/1848368/42002360-eb922e38-7a1b-11e8-8d16-72327d3f0ac7.png)

**Comment by jeromeetienne** [comment]
i spent 10 months coding on this fulltime for free but i was unemployed at the time. At some point, i needed to be able to pay my rent, so i looked for a job. i got no shame on what i done. 

If anybody is volonteering to actually produce the work to help the project, please please contact me.

PS: please do so contructively. Honnestly i dont understand what @Utopiah is doing, he is complaining about the project handling, but he is saying he got no interest in it. and dont want to put the needed work.

**Comment by jeromeetienne** [comment]
oh and for people willing to handle opensource projects, be aware you will receive this kind of tweet in the morning :) [tweet](https://twitter.com/utopiah/status/960823123431051264)

Overwhemling people are nice with open-source authors, but they are others...

Stay conscious that people spent a lot of time providing good work for free. so please be nice with them, it makes their life easier.

**Comment by Utopiah** [comment]
@jeromeetienne this kind of tweet come from a place of concern for your work precisely, from the efforts you put it and that should not be lost. It does not place from a place of criticism. Yet at some point when you do make an open source project but you do not have the ability to maintain it for one way or another I do feel the best thing for the project is to let others who are offering and have showcase the ability to maintain it to do give them push access. If for some reason, whatever it is, you can not do that then it's good to communicate about it just in the read me of the project or simply to answer the call for help that people are having. I wasn't the only one asking for an answer, any answer.

This wasn't but a random criticism on best practices but a call for update, any update, BECAUSE the project does fill a need some so I'm grateful for that. I tried to reach you by email and received no answer. Others have tried to reach you by email before that weeks ago, also no answer. Even though it is a useful project and yes you did do the work, nobody is questioning that, if nothing is done it will die down regardless of the circumstances that lead to it. That's your decision also.

**Comment by Utopiah** [comment]
To be practical when @jeromeetienne you say "If anybody is volonteering to actually produce the work to help the project, please please contact me." but @nicolocarpignoli precisely did so more than once month ago "I can offer myself for a little help of course: I know this is work for free and I'm offering for this, also hoping someone wants to help too." what is the problem?

**Comment by TimVanMourik** [comment]
Hi @jeromeetienn, @Utopiah and others. I think the initial issue wasn’t meant as an accusation (despite the aggressive title), but it can easily feel that way when some acknowledgment/gratitude for tremendous body of work isn’t expressed. So first of, I’d like to mention that I’m increbily grateful to @jeromeetienne to make it so easy for me to right my first AR app. 

Second, it’s a recurring problem that [funding OSS is hard](https://hackernoon.com/why-funding-open-source-is-hard-652b7055569d), and low-reward. There’s little appreciation and there’s no shame in wanting to pay bills at some point. Plus there’s the developer’s curse that the more popular a repo becomes, the more (perceived) obligation it creates for the maintainer.

But more on topic, I think it’s important to find a working soluion (Yay for @nicolocarpignoli to offer some help). I am happy to document the problems we encountered in developing our app and fixing them in the code basis where possible, but I’m completey new to AR and my JS skills are limited, so I won’t be of much use. And I’ve got some OSS repos of myself to tend to. 

So thanks @Utopiah for raising this point and let’s try and be civil in finding a working solution 🙂

**Comment by jeromeetienne** [comment]
@nicolocarpignoli excelent, i definitly could use your help. Thanks for proposing, lets talk offline, maybe a conf call to syncup ?

when i said "putting the actual work", i didnt imply coding. it was a comment on people who spent their time criticizing instead of helping. and those people are not my cup of tea :)

**Comment by gravyplaya** [comment]
I would also like to volunteer to help keep this going. Whats the best way? Make a fork and point everyone to the fork?

**Comment by jeromeetienne** [comment]
What would like to do to help?

**Comment by stemkoski** [comment]
As @nicolocarpignoli mentioned above, no single person can replace @jeromeetienne in this project, and hopefully many of us can find a way to contribute to this effort. For my part, I have been working on a set of examples based on Jerome's work, over at https://github.com/stemkoski/AR-Examples. Since I am an educator, the purpose of my repository is to create a set of examples for people to learn from, staring from simple things, building up in complexity to the amazing examples Jerome has displayed in the past.

I think the most important step at this point is to update the project Readme or enable the project Wiki, to make it easier for newcomers to get started with this awesome library. I don't have time to add new features either, but I can volunteer to help with documentation.

**Comment by nicolocarpignoli** [comment]
@stemkoski I totally agree with you, a good Readme is the first step. We can also provide a wiki with much more rich docs and link it on the readme. As soon as I can catch Jerome by chat I will suggest it to him :)

**Comment by nicolocarpignoli** [comment]
@stemkoski and @gravyplaya and others who wanted to help, we have created a slack channel for collaborators. there we can chat and see how each of us can help and what contribute we can give :)

see @https://github.com/jeromeetienne/AR.js/issues/376

if you wanted to help, please write to me or @jeromeetienne your email privately and we will add you to slack channel

**Comment by CueContext** [comment]
New to this forum and more than I should write here:

I have taken a deep dive into the work that @jeromeetienne and others here have created here. This is amazing work, innovative, and done with passion. 

I agree with @nicolocarpignoli. This project could be greatly aided by a ReadMe or Wiki that defines a few near-term goals, desired longer-term outcomes, and status, so that: 1) newcomers can see and follow what's going on, and 2) contributors can work on discrete areas toward these goals according to their skills and interests. I would defer to @jeromeetienne to set the vision and key milestones (if you have time given your new focus), but others who have spent time working with this code base could suggest tactical and strategic goals as candidate ideas for a "Draft Vision" that @jeromeetienne could shape and finalize.  

I am glad to volunteer to help if I can. 

TL;DR!

Why do this?
For me, the essential purpose of this kind of technology is less about entertainment and gaming, and more about communication, inspiration, connection, and intimacy (from facilitating hard-to-grasp subject matter to having shared-experiences with others).  

I have young kids. I don't want them on devices all day. Ironically, AR can get kids interested in the real world again. AR, if easily accessible via the mobile web and done with passion, can allow anyone to see the real world in new ways that trigger emotions and ideas that are truly unique and special. 

I am interested in this project because I am working on new types of digital educational engagement solutions for zoos and visitor-based organizations, and I believe mobile web AR is just in time.  The biggest challenge for me is not so much the technology, but rather getting people who lead these types of organizations to: 1) embrace new ways of engaging the public as part of their mission, and 2) adopt new trans-media storytelling strategies for reaching and communicating with the public. 

**Aside from sharing technology issues, sharing ideas on how people are using this technology to make changes in the real world would be an equally valuable forum.**
